Source/WebCore/platform/graphics/cocoa/VP9UtilitiesCocoa.mm

```diff
@@ -155,23 +145,12 @@ bool isVP9DecoderAvailable()
         return true;
 
 #if PLATFORM(IOS) || PLATFORM(VISION)
-    return canLoad_VideoToolbox_VTIsHardwareDecodeSupported() && VTIsHardwareDecodeSupported(kCMVideoCodecType_VP9);
+    return vp9HardwareDecoderAvailable();
 #else
-    if (!VideoToolboxLibrary(true))
-        return false;
-    return noErr == VTSelectAndCreateVideoDecoderInstance(kCMVideoCodecType_VP9, kCFAllocatorDefault, nullptr, nullptr);
+    return true;
```

Comment:
When libwebrtc is not available, we do not have access to libvpx.
We should keep those checks and if not successful, check for libwebrtc being available.

Reply:
then decoding will error.

Reply:
I've amended the test, now checking if the VideoDecoder supports vp8 or vp9